locales/zh_CN/LC_MESSAGES/scalar_tensor.po

```diff
@@ -93,14 +94,15 @@ msgstr ""
 #: ../../scalar_tensor.rst:44
 #, fuzzy
 msgid ""
-"Not providing ``shape`` allows you to *place* the tensor in a layout "
-"other than the default *dense*, see :ref:`layout` for more details."
-msgstr "不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，见 :ref:`sparse` 。"
+"Not providing ``shape`` allows you to *place* the tensor in a layout other than "
+"the default *dense*, see :ref:`layout` for more details."
+msgstr ""
+"不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，见 :ref:`sparse` 。"
```

Comment:
needs help here :hear_no_evil: Someone please corrects me if I'm wrong. Is the link for `sparse` or `layout`? 
I checked the latest version it seems like the sparse is still WIP?
```suggestion
"不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，参考见 :ref:`layout` 。"
```

Reply:
wow,It's my first time to use "pull request".I don't know whether I did it right.
Should I modify the part you mentioned above and then merge again?

Reply:
> wow,It's my first time to use "pull request".I don't know whether I did it right.
> Should I modify the part you mentioned above and then merge again?

You could directly apply suggested changes here; After that, it should show suggestion resolved; After others review your pull request, we can merge it to master :-)